emva1288/process/plotting.py

```diff
@@ -510,7 +510,7 @@ class PlotLogarithmicHistogramDSNU(Emva1288Plot):
     '''Create Logarithmic histogram DSNU plot'''
 
     name = 'Logarithmic histogram DSNU'
-    xlabel = 'Dark value [DN]'
+    xlabel = 'Dark value [e^{-}]'
```

Comment:
`test.DSNU1288` is not used in this plot, so the units haven't changed.

Reply:
okay, what is this supposed to be? is that DN?